stock_inventory_revaluation/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Matt Taylor
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Inventory Revaluation",
+    "summary": "Introduces inventory revaluation as single point to change "
+               "the valuation of products.",
+    "version": "12.0.1.0.0",
+    "author": "Eficent Business and IT Consulting Services S.L., "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Matt Taylor,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.eficent.com",
```

Comment:
Please change this according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Reply:
Done.  Thank you.